M1 macOS编译报错

**1. 环境（environment）**
 - Build OS and Version: MacBook Pro (13-inch, M1, 2020), macOS 12.4
 - RunTime OS Version: MacBook Pro (13-inch, M1, 2020), macOS 12.4
 - RunTime DEVICE: ARM / Metal / Apple NPU

  **2. Github版本**
 - branch: master
 - commit(optional): 最新

  **3. 编译方式(compile method)**
  ```shell
#!/bin/bash
CC=`which clang`
CXX=`which clang++`
cmake ../../TNN \
    -DCMAKE_OSX_SYSROOT=macosx \
    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 \
    -DCMAKE_SYSTEM_PROCESSOR=aarch64 \
    -DCMAKE_C_COMPILER=$CC \
    -DCMAKE_CXX_COMPILER=$CXX \
    -DTNN_CPU_ENABLE=ON \
    -DTNN_ARM_ENABLE=ON \
    -DTNN_ARM82_ENABLE=ON \
    -DTNN_METAL_ENABLE=ON \
    -DTNN_OPENCL_ENABLE=OFF \
    -DTNN_APPLE_NPU_ENABLE=ON \
    -DTNN_BUILD_SHARED=OFF \
    -DTNN_TEST_ENABLE=OFF \
    -DTNN_UNIT_TEST_ENABLE=OFF \
    -DTNN_QUANTIZATION_ENABLE=OFF \
    -DTNN_MODEL_CHECK_ENABLE=OFF \
    -DDEBUG=OFF \
    -DTNN_PROFILER_ENABLE=OFF \
    -DTNN_BENCHMARK_MODE=OFF \
    -DTNN_UNIT_TEST_BENCHMARK=OFF \
    -DTNN_CONVERTER_ENABLE=OFF \
    -DTNN_ONNX2TNN_ENABLE=OFF \
    -DTNN_TNN2MEM_ENABLE=OFF \
    -DTNN_BUILD_BENCHMARK_TEST_LIB_ENABLE=OFF \
    -DTNN_METAL_FLOAT32=ON \
    -DTNN_COREML_FLOAT32=ON \
    -DTNN_DYNAMIC_RANGE_QUANTIZATION_ENABLE=OFF
make -j8
```

  **4. 编译日志(build log)**
 ```txt
-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The ASM compiler identification is Clang with GNU-like command-line
-- Found assembler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
/Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN
/Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN
-- >>>>>>>>>>>>>
-- TNN BUILD INFO:
-- 	System: Darwin
-- 	Processor: arm64
-- 	Cpu:	ON
-- 	X86:	OFF
-- 	Arm:	ON
-- 	Arm82:	ON
-- 	Metal:	ON
-- 	OpenCL:	OFF
-- 	CUDA:	OFF
-- 	DSP:	OFF
-- 	Atlas:	OFF
-- 	TensorRT:	OFF
-- 	AppleNPU:	ON
-- 	HuaweiNPU:	OFF
-- 	RKNPU:	OFF
-- 	OpenVINO:	OFF
-- 	OpenMP:	OFF
-- 	TEST:	OFF
-- 	--Unit Test:	OFF
-- 	Quantization:	OFF
-- 	ModelCheck:	OFF
-- 	DEBUG:	OFF
-- 	PROFILE:	OFF
-- 	BENCHMARK:	OFF
-- 	BENCHMARK Layer:	OFF
-- 	Model Converter:	OFF
-- 	ONNX2TNN Converter:	OFF
-- 	TNN2MEM:	OFF
-- 	BENCHMARK Test Lib:	OFF
-- 	Dynamic Range Quantization:	OFF
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/builds/macos-static-arm64
+ make -j8
Scanning dependencies of target TNNArm82
Scanning dependencies of target TNNArm
[  0%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv_layer_1x1.mm.o
[  0%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv1d_layer_acc.mm.o
[  0%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv_layer_acc.mm.o
[  1%] Compile metal shaderes: fp32
[  1%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_batch_norm_fp16_layer.cc.o
[  1%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/compute/compute_elewise.cc.o
[  1%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/coreml_model.mm.o
[  1%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_abs_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/convolution/metal_conv_layer_1x1.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/convolution/metal_conv_layer_common.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/convolution/metal_conv_layer_depthwise.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/convolution/metal_conv_layer_winograd.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/convolution/metal_inner_product_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/deconvolution/metal_deconv_layer_common.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/deconvolution/metal_deconv_layer_depthwise.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_abs_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_acos_layer_acc.metal
[  2%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_binary_fp16_layer_acc.cc.o
[  2%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/coreml_network.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_add_layer_acc.metal
[  2%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_add_layer_acc.cc.o
[  2%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/compute/compute_int8.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_arg_max_or_min_layer_acc.metal
[  2%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_add_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_asin_layer_acc.metal
[  2%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_arg_max_or_min_layer_acc.cc.o
[  2%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv_layer_common.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_atan_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_batch_norm_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_cast_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_ceil_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_clip_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_common.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_concat_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_cos_layer_acc.metal
[  2%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu__histogram_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_div_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_elu_layer_acc.metal
[  2%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_batch_norm_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_exp_layer_acc.metal
[  2%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_fp16_layer_3x3.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_floor_layer_acc.metal
[  2%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv_layer_depthwise.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_gather_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_gelu_layer_acc.metal
[  3%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_binary_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_grid_sample_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_hard_sigmoid_layer_acc.metal
[  3%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_fp16_layer_c3.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_hard_swish_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_hdrguide_layer_acc.metal
[  3%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_abs_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_instance_norm_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_inverse_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_layer_norm_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_log_layer_acc.metal
[  3%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_base_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_log_sigmoid_layer_acc.metal
[  4%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_conv_layer_winograd.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_lrn_layer_acc.metal
[  5%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_batchnorm_layer.cc.o
[  5%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/convolution/metal_inner_product_layer_acc.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_mat_mul_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_max_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_min_layer_acc.metal
[  5%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_fp16_layer_common.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_mul_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_acos_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_neg_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_normalize_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_pad_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_padv2_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_permute_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_pixel_shuffle_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_pooling_layer_acc.metal
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_binary_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_pow_layer_acc.metal
[  6%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_cast_layer_acc.cc.o
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_cast_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_prelu_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_prior_box_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_add_layer_acc.cc.o
[  6%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/deconvolution/metal_deconv_layer_acc.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_reciprocal_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_reduce_layer_acc.metal
[  6%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_fp16_layer_depthwise.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_reformat_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_relu6_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_relu_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_reorg_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_reshape_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_selu_layer_acc.metal
[  6%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_clip_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_shuffle_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_and_layer_acc.cc.o
[  6%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_concat_layer_acc.cc.o
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_clip_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_sigmoid_layer_acc.metal
[  6%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_fp16_layer_depthwise_s1.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_sign_layer_acc.metal
[  6%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/deconvolution/metal_deconv_layer_common.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_signed_mul_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_arg_max_or_min_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_sin_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_softmax_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_softplus_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_splitv_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_sqrt_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_squeeze_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_stride_slice_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_sub_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_asin_layer_acc.cc.o
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_concat_layer.cc.o
[  6%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_int8_sdot_layer_common.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_tan_layer_acc.metal
[  6%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_detection_output_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_tanh_layer_acc.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_atan_layer_acc.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_tile_layer_acc.metal
[  6%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/deconvolution/metal_deconv_layer_depthwise.mm.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/metal_upsample_layer_acc.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/acc/recurrent/metal_lstm_layer_acc.metal
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_const_layer.cc.o
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/metal_blob_converter.metal
       Compile /Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/metal_mat_converter.metal
[  6%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_batch_norm_layer_acc.cc.o
[  6%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_div_layer_acc.cc.o
[  6%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_conv_layer.cc.o
[  6%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_abs_layer_acc.mm.o
[  7%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_conv_int8_sdot_layer_depthwise_3x3.cc.o
[  7%] Built target tnn.metallib
[  8%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_bias_add_layer_acc.cc.o
[  8%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_elu_layer_acc.cc.o
[  8%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_acos_layer_acc.mm.o
[  8%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_deconv_layer.cc.o
[  8%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_binary_op_layer_acc.cc.o
[  8%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_exp_layer_acc.cc.o
[  8%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_deconv_fp16_layer_common.cc.o
[  8%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_bitshift_layer_acc.cc.o
[  9%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_div_layer.cc.o
[  9%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_add_layer_acc.mm.o
[  9%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_deconv_fp16_layer_depthwise.cc.o
[ 10%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_expand_layer_acc.cc.o
[ 10%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_cast_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_arg_max_or_min_layer_acc.mm.o
[ 11%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_inner_product_fp16_layer.cc.o
[ 11%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_floor_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_flatten_layer.cc.o
[ 11%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_ceil_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_asin_layer_acc.mm.o
[ 11%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_gather_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_instance_norm_fp16_layer.cc.o
[ 11%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_gather_layer.cc.o
[ 11%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_clip_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_atan_layer_acc.mm.o
[ 11%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_gelu_layer.acc.cc.o
[ 11%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_layer_norm_fp16_layer.cc.o
[ 11%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_concat_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_batch_norm_layer_acc.mm.o
[ 11%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_grid_sample_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_lstm_fp16_layer.cc.o
[ 11%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_gelu_layer.cc.o
[ 11%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_const_layer_acc.cc.o
[ 11%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_hard_sigmoid_acc.cc.o
[ 11%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_cast_layer_acc.mm.o
[ 12%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_constantofshape_layer_acc.cc.o
[ 12%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_hard_swish_acc.cc.o
[ 12%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_prelu_fp16_layer.cc.o
[ 12%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_hard_sigmoid_layer.cc.o
[ 12%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_ceil_layer_acc.mm.o
[ 12%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_hard_swish_layer.cc.o
[ 13%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_inner_product_layer_acc.cc.o
[ 13%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_conv_1d_layer_acc.cc.o
[ 13%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_clip_layer_acc.mm.o
[ 14%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_relu6_fp16_layer.cc.o
[ 14%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_instance_norm_layer_acc.cc.o
[ 14%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_innerproduct_layer.cc.o
[ 14%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_concat_layer_acc.mm.o
[ 14%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_conv_3d_layer_acc.cc.o
[ 14%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_relu_fp16_layer.cc.o
[ 15%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_cos_layer_acc.mm.o
[ 15%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_sigmoid_fp16_layer.cc.o
[ 15%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_inverse_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_conv_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_layernorm_layer.cc.o
[ 15%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/arm_softmax_fp16_layer.cc.o
[ 15%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_cpu_adapter_acc.mm.o
[ 15%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_layer_norm_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/compute_half.cc.o
[ 15%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_log_acc_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_cos_layer_acc.cc.o
[ 15%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_log_sigmoid_layer_acc.cc.o
[ 16%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_mat_mul_layer.cc.o
[ 16%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/compute_sdot_int8.cc.o
[ 16%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_div_layer_acc.mm.o
[ 16%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_deconv_layer_acc.cc.o
[ 16%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_lstm_layer_acc.cc.o
[ 16%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/gemm_function_fp16.cc.o
[ 16%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_detection_output_layer_acc.cc.o
[ 17%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_mat_mul_layer_acc.cc.o
[ 17%] Building CXX object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/winograd_function_fp16.cc.o
[ 17%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_mul_layer.cc.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/CONV_DW_3X3S1_INT8_SDOT_SLIDEW.S.o
[ 18%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_max_layer_acc.cc.o
[ 18%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_pad_layer.cc.o
[ 18%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_detection_post_process_layer_acc.cc.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/CONV_DW_3X3S2_INT8_SDOT_SLIDEW.S.o
[ 18%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_elu_layer_acc.mm.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/CONV_DW_3X3_FP16_SLIDEW.S.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/CONV_FP16_SLIDEW_C3.S.o
[ 18%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_padv2_layer.cc.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/DECONV_FP16_O8.S.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/DECONV_FP16_O8_C1.S.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/FLOAT2HALF.S.o
[ 18%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/GEMM_FP16_N8.S.o
[ 19%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/GEMM_INT8_SDOT_8X4.S.o
[ 19%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/GEMM_INT8_SDOT_8X8.S.o
[ 19%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/GEMV_INT8_SDOT.S.o
[ 19%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_min_layer_acc.cc.o
[ 19%] Building ASM object source/tnn/device/arm/acc/compute_arm82/CMakeFiles/TNNArm82.dir/asm_64/HALF2FLOAT.S.o
[ 19%] Built target TNNArm82
[ 20%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_div_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_mul_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_nchw_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_permute_layer.cc.o
[ 20%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_exp_layer_acc.mm.o
[ 20%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_flatten_layer_acc.mm.o
[ 20%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_einsum_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_neg_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_elu_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_equal_layer_acc.cc.o
[ 20%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_floor_layer_acc.mm.o
[ 20%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_pool_layer.cc.o
[ 20%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_erf_layer_acc.cc.o
[ 21%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_gather_layer_acc.mm.o
[ 21%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_normalize_layer_acc.cc.o
[ 21%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_prelu_layer.cc.o
[ 21%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_gelu_layer_acc.mm.o
[ 21%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_exp_layer_acc.cc.o
[ 21%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_pad_layer_acc.cc.o
[ 21%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_expand_layer_acc.cc.o
[ 21%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_reduce_max_layer.cc.o
[ 21%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_flatten_layer_acc.cc.o
[ 22%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_padv2_layer_acc.cc.o
[ 23%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_reduce_mean_layer.cc.o
[ 23%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_grid_sample_layer_acc.mm.o
[ 23%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_permute_layer_acc.cc.o
[ 23%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_reduce_min_layer.cc.o
[ 23%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_pixel_shuffle_layer_acc.cc.o
[ 23%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_hard_sigmoid_layer_acc.mm.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_floor_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_pool_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_hard_swish_layer_acc.mm.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_gather_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_hdrguide_layer_acc.mm.o
[ 24%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_reduce_sum_layer.cc.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_gathernd_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_gelu_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_pow_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_prelu_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_greater_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_instance_norm_layer_acc.mm.o
[ 24%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_relu6_layer.cc.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_gridsample_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_priorbox_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_inverse_layer_acc.mm.o
[ 24%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reciprocal_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_group_norm_layer_acc.cc.o
[ 24%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_relu_layer.cc.o
[ 25%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_l1_layer_acc.cc.o
[ 26%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_layer_acc.mm.o
[ 26%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_l2_layer_acc.cc.o
[ 26%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_reshape_layer.cc.o
[ 26%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_hard_sigmoid_layer_acc.cc.o
[ 26%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_layer_acc.cc.o
[ 26%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_layer_norm_layer_acc.mm.o
[ 26%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_shape_layer.cc.o
[ 26%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_log_sum_exp_layer_acc.cc.o
[ 27%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_hard_swish_layer_acc.cc.o
[ 27%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_hdrguide_layer_acc.cc.o
[ 27%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_log_sum_layer_acc.cc.o
[ 27%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_log_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_shuffle_channel_layer.cc.o
[ 28%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_log_sigmoid_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_sigmoid_layer.cc.o
[ 28%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_max_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_lrn_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_inner_product_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_slice_layer.cc.o
[ 28%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_instance_norm_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_mean_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_mat_mul_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_inverse_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_max_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_min_layer_acc.cc.o
[ 28%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_slice_v2_layer.cc.o
[ 28%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_min_layer_acc.mm.o
[ 28%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_softmax_layer.cc.o
[ 28%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_layer_acc.cc.o
[ 29%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_prod_layer_acc.cc.o
[ 29%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_sum_layer_acc.cc.o
[ 30%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_multidir_broadcast_layer_acc.mm.o
[ 30%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_mul_layer_acc.mm.o
[ 30%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_layer_norm_layer_acc.cc.o
[ 30%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_softplus_layer.cc.o
[ 30%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reduce_sum_square_layer_acc.cc.o
[ 30%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_neg_layer_acc.mm.o
[ 30%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reformat_layer_acc.cc.o
[ 30%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_relu6_layer_acc.cc.o
[ 31%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_less_layer_acc.cc.o
[ 31%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_softsign_layer.cc.o
[ 31%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_relu_layer_acc.cc.o
[ 31%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_normalize_layer_acc.mm.o
[ 31%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reorg_layer_acc.cc.o
[ 31%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_log_layer_acc.cc.o
[ 31%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_pad_layer_acc.mm.o
[ 31%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_split_layer.cc.o
[ 32%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_reshape_layer_acc.cc.o
[ 32%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_log_sigmoid_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_squeeze_layer.cc.o
[ 33%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_log_softmax_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_roialign_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_sub_layer.cc.o
[ 33%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_tanh_layer.cc.o
[ 33%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_padv2_layer_acc.mm.o
[ 33%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_lrn_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_scale_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_permute_layer_acc.mm.o
[ 33%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_unsqueeze_layer.cc.o
[ 33%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_lstm_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_selu_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_mat_mul_layer_acc.cc.o
[ 33%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_pixel_shuffle_layer_acc.mm.o
[ 33%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_shuffle_layer_acc.cc.o
[ 34%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_pooling_layer_acc.mm.o
[ 34%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/layer_convert/coreml_upsample_layer.cc.o
[ 34%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_sigmoid_layer_acc.cc.o
[ 34%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_pow_layer_acc.mm.o
[ 34%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_max_layer_acc.cc.o
[ 34%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/DataStructures.pb-c.c.o
[ 34%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/FeatureTypes.pb-c.c.o
[ 34%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_sign_layer_acc.cc.o
[ 34%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/Model.pb-c.c.o
[ 34%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_signed_mul_layer_acc.cc.o
[ 35%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_min_layer_acc.cc.o
[ 36%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_softmax_layer_acc.cc.o
[ 36%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_mul_layer_acc.cc.o
[ 37%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/NeuralNetwork.pb-c.c.o
[ 37%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/Parameters.pb-c.c.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_softplus_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_prelu_layer_acc.mm.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_splitv_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_neg_layer_acc.cc.o
[ 37%] Building C object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/mlmodel/src/protobuf-c.c.o
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_prior_box_layer_acc.mm.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_sqrt_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reciprocal_layer_acc.mm.o
[ 37%] Building CXX object source/tnn/network/coreml/CMakeFiles/TNNAppleNPU.dir/tnn_impl_coreml.mm.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_non_max_suppression.cc.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_nonzero_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_squeeze_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_l1_layer_acc.mm.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_normalize_layer_acc.cc.o
[ 37%] Built target TNNAppleNPU
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_l2_layer_acc.mm.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_stride_slice_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_not_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_sub_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_onehot_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_tile_layer_acc.cc.o
[ 37%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_layer_acc.mm.o
[ 38%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pad_layer_acc.cc.o
[ 38%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_padv2_layer_acc.cc.o
[ 39%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_trig_layer_acc.cc.o
[ 39%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_unary_layer_acc.cc.o
[ 39%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_permute_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_log_sum_exp_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pixel_shuffle_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_log_sum_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_unsqueeze_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/arm_upsample_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_max_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_mean_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/binary_function.cc.o
[ 40%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pool_1d_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pool_3d_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_min_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pool_layer_acc.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_prod_layer_acc.mm.o
[ 40%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/compute.cc.o
[ 40%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_sum_layer_acc.mm.o
[ 41%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reduce_sum_square_layer_acc.mm.o
[ 41%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_pow_layer_acc.cc.o
[ 42%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_prelu_layer_acc.cc.o
[ 42%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_prior_box_layer_acc.cc.o
[ 42%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reformat_layer_acc.mm.o
[ 42%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_range_layer_acc.cc.o
[ 42%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/compute_int8.cc.o
[ 42%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reciprocal_layer_acc.cc.o
[ 42%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/gemm_function.cc.o
[ 42%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_relu6_layer_acc.mm.o
[ 43%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/winograd_function.cc.o
[ 43%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_l1_layer_acc.cc.o
[ 43%] Building CXX object source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/convolution/arm_conv1d_layer_acc.cc.o
/Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/arm/acc/compute/gemm_function.cc:724:17: error: macro 'INIT1x8' is already defined
                ".macro INIT1x8                     \n"
                ^
<inline asm>:1:2: note: instantiated into assembly here
        .macro INIT1x8
        ^
/Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/arm/acc/compute/gemm_function.cc:732:18: error: macro 'SAVE1x8' is already defined
                ".macro SAVE1x8                     \n"
                 ^
<inline asm>:9:1: note: instantiated into assembly here
.macro SAVE1x8
^
2 errors generated.
make[2]: *** [source/tnn/device/arm/CMakeFiles/TNNArm.dir/acc/compute/gemm_function.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 43%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_relu_layer_acc.mm.o
[ 43%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reorg_layer_acc.mm.o
[ 43%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_reshape_layer_acc.mm.o
[ 43%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_selu_layer_acc.mm.o
[ 43%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_l2_layer_acc.cc.o
[ 43%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_layer_acc.cc.o
[ 43%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_shuffle_layer_acc.mm.o
[ 44%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_sigmoid_layer_acc.mm.o
make[1]: *** [source/tnn/device/arm/CMakeFiles/TNNArm.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_log_sum_exp_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_log_sum_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_sign_layer_acc.mm.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_max_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_mean_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_min_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_prod_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_sum_square_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reduce_sum_layer_acc.cc.o
[ 45%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_signed_mul_layer_acc.mm.o
[ 45%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_sin_layer_acc.mm.o
[ 46%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reformat_layer_acc.cc.o
[ 46%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_softmax_layer_acc.mm.o
[ 46%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_softplus_layer_acc.mm.o
[ 46%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_relu6_layer_acc.cc.o
[ 46%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_splitv_layer_acc.mm.o
[ 46%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_sqrt_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_squeeze_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_relu_layer_acc.cc.o
[ 47%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reorg_layer_acc.cc.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_stride_slice_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_stride_slice_v2_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_sub_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_tan_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_tanh_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_reshape_layer_acc.cc.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_tile_layer_acc.mm.o
[ 47%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_roialign_layer_acc.cc.o
[ 47%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_unary_layer_acc.mm.o
[ 48%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/metal_upsample_layer_acc.mm.o
[ 48%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_rsqrt_layer_acc.cc.o
[ 48%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/acc/recurrent/metal_lstm_layer_acc.mm.o
[ 48%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/metal_blob_converter.mm.o
[ 48%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_scale_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_scatter_elements_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/metal_command_queue.mm.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_scatter_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/metal_context.mm.o
[ 49%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/metal_device.mm.o
/Users/Smeegol/Git/FFmpeg-AI-Filters-Dev/TNN-Dev/TNN/source/tnn/device/metal/metal_command_queue.mm:19:17: warning: method 'commandBufferWithDescriptor:' in protocol 'MTLCommandQueue' not implemented [-Wprotocol]
@implementation TNNMetalCommandQueueImpl
                ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk/System/Library/Frameworks/Metal.framework/Headers/MTLCommandQueue.h:40:1: note: method 'commandBufferWithDescriptor:' declared here
- (nullable id <MTLCommandBuffer>)commandBufferWithDescriptor:(MTLCommandBufferDescriptor*)descriptor API_AVAILABLE(macos(11.0), ios(14.0));
^
1 warning generated.
[ 49%] Building CXX object source/tnn/device/metal/CMakeFiles/TNNMetal.dir/metal_mat_converter.mm.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_scatter_nd_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_selu_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_shape_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_shuffle_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_sigmoid_layer_acc.cc.o
[ 49%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_sign_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_signed_mul_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_sin_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_size_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_softmax_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_softplus_layer_acc.cc.o
[ 50%] Built target TNNMetal
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_splitv_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_sqrt_layer_acc.cc.o
[ 50%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_squared_difference_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_squeeze_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_stride_slice_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_stride_slice_v2_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_sub_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_swish_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_tan_layer_acc.cc.o
[ 51%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_tanh_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_tile_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_topk_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_unary_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_unsqueeze_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_upsample_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/acc/cpu_where_layer_acc.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/cpu_blob_converter.cc.o
[ 52%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/cpu_context.cc.o
[ 53%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/cpu_device.cc.o
[ 53%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/cpu_mat_converter.cc.o
[ 53%] Building CXX object source/tnn/device/cpu/CMakeFiles/TNNCpu.dir/cpu_mat_util.cc.o
[ 53%] Built target TNNCpu
make: *** [all] Error 2
 ```

  **5. 详细描述bug 情况 (Describe the bug)**


  **6. 运行日志（runtime log）**

  **7. 截图（Screenshots）**
<img width="1393" alt="image" src="https://user-images.githubusercontent.com/1212201/177475710-4e2a1cbc-b461-47d8-8ad0-6a2d250626f9.png">

--- a/source/tnn/device/arm/acc/compute/gemm_function.cc
+++ b/source/tnn/device/arm/acc/compute/gemm_function.cc
@@ -720,26 +720,24 @@ void Kernel_1x8(int m, int n, int k, const float *sa, const float *sb, float *sc
 #ifdef __aarch64__
             int64_t ldc_offset = ldc * sizeof(float) - 16;
             int64_t k_64       = k;
-            asm volatile(
-                ".macro INIT1x8                     \n"
-                "   mov x9,        %2               \n"
-                "   ld1 {v8.4s},  [x9], #16         \n"
-                "   ld1 {v20.4s}, [x9], %3          \n"
-                "   movi v9.4s,    #0               \n"
+
+            #define INIT1x8                             \
+                "   mov x9,        %2               \n" \
+                "   ld1 {v8.4s},  [x9], #16         \n" \
+                "   ld1 {v20.4s}, [x9], %3          \n" \
+                "   movi v9.4s,    #0               \n" \
                 "   movi v21.4s,   #0               \n"
-                ".endm                              \n"
-                "                                   \n"
-                ".macro SAVE1x8                     \n"
-                "   mov x9,        %2               \n"
-                "   fadd v8.4s,  v8.4s,  v9.4s      \n"
-                "   fadd v20.4s, v20.4s, v21.4s     \n"
-                "   st1 {v8.4s},  [x9], #16         \n"
-                "   st1 {v20.4s}, [x9], %3          \n"
-                ".endm                              \n"
-                "                                   \n"
+            #define SAVE1x8                             \
+                "   mov x9,        %2               \n" \
+                "   fadd v8.4s,  v8.4s,  v9.4s      \n" \
+                "   fadd v20.4s, v20.4s, v21.4s     \n" \
+                "   st1 {v8.4s},  [x9], #16         \n" \
+                "   st1 {v20.4s}, [x9], %3          \n" \
+
+            asm volatile(
                 "   ld1 {v0.4s}, [%0], #16          \n"
                 "   ld1 {v2.4s}, [%1], #16          \n"
-                "INIT1x8                            \n"
+                INIT1x8
                 "mov x8,%4                          \n"
                 "0:                                 \n"
                 "   subs x9, x8, #4                 \n"
@@ -775,11 +773,14 @@ void Kernel_1x8(int m, int n, int k, const float *sa, const float *sb, float *sc
                 "   ld1 {v2.4s}, [%1], #16          \n"
                 "   bne 1b                          \n"
                 "2:                                 \n"
-                "   SAVE1x8                         \n"
+                    SAVE1x8
                 "                                   \n"
                 : "=r"(b), "=r"(a), "=r"(c), "=r"(ldc_offset), "=r"(k_64)
                 : "0"(b), "1"(a), "2"(c), "3"(ldc_offset), "4"(k_64)
                 : "memory", "cc", "x8", "x9", "v0", "v1", "v2", "v3", "v4", "v8", "v9", "v20", "v21");
+
+            #undef INIT1x8
+            #undef SAVE1x8
 #else
             int ldc_offset = ldc * sizeof(float) - 16;
             asm volatile(

